client/src/Questions/QuestionForm.js

```diff
@@ -9,9 +9,12 @@ import ValidationError from '../ValidationError';
 
 function QuestionForm() {
   const { id } = useParams();
+  const query = new URLSearchParams(useLocation().search);
   const history = useHistory();
   const [error, setError] = useState(null);
+  const [questionnaires, setQuestionnaires] = useState([]);
   const [question, setQuestion] = useState({
+    QuestionnaireId: query.get('QuestionnaireId') || '',
```

Comment:
We check the query params for a QuestionnaireId, and set it if we have it, OR, if it is undefined, we set an empty string